pkgs/by-name/hy/hydrapaper/package.nix

```diff
@@ -0,0 +1,78 @@
+{ lib,
+  stdenv,
+  fetchFromGitLab,
+  meson,
```

Comment:
I would recommend that you sort them in the order they are used, but this is not necessary